tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -305,6 +364,20 @@ Object.PCM {
 			formats	'S16_LE,S24_LE,S32_LE'
 		}
 	}
+	pcm."18" {
+		name	"DMIC Raw Stream"
```

Comment:
DMIC Raw? No need to add a 'Stream" qualifier for a PCM device

Reply:
> DMIC Raw? No need to add a 'Stream" qualifier for a PCM device

done